galette/lib/Galette/Entity/Group.php

```diff
@@ -248,14 +248,6 @@ private function loadSubGroups(): void
         try {
             $select = $zdb->select(self::TABLE, 'a');
 
-            if (!$this->login->isAdmin() && !$this->login->isStaff()) {
-                $select->join(
-                    ['b' => PREFIX_DB . self::GROUPSMANAGERS_TABLE],
-                    'a.' . self::PK . '=b.' . self::PK,
-                    []
-                )->where(['b.' . Adherent::PK => $this->login->id]);
-            }
-
```

Comment:
Are we sure this should be removed?

Reply:
Currently, on the `develop` branch, this select seems to be "faulty" as all the subgroups managed by a groupmanager that are not subgroups of a parent group also managed by the groupmanager can't be displayed in the tree.

So I choose to completely remove it in this PR and display the full tree to groupmanagers. Maybe this select can be fixed to properly display all the groups actually managed, but I wasn't able to do it.